src/components/inspectors/mixins/DataFormat.js

```diff
@@ -0,0 +1,112 @@
+import Validator from 'validatorjs';
```

Comment:
This file, along with `validation.js`, were made to handle all form renderer components, but since we're only dealing with the date picker in this case, I think they can be significantly reduced and possibly inlined into the datepicker component.